h/static/scripts/test/time-test.js

```diff
@@ -50,6 +51,34 @@ describe('time', function () {
   });
 
   describe('.toFuzzyString', function () {
+
+    var originalIntl;
+
+    before(function () {
+
+      // Intl isn't implemented in PhantomJS so we have to mock it in these
+      // tests. The mock version returns the results we'd expect from the real
+      // one for the test values that we use.
+      originalIntl = window.Intl;
```

Comment:
I don't think either 2 or 3 adds much in this case so would go for 1.

But what result are we hoping for from the tests here? If we add an optional `Intl` argument there'll be four possible combinations of the two `Intl` values:

1. The test pass in a mock `Intl` and `window.Intl` is defined (e.g. we're running the tests in Chrome) - it should use th